src/cloud-providers/package.json

```diff
@@ -30,6 +30,7 @@
     "umd"
   ],
   "dependencies": {
+    "@auth0/auth0-spa-js": "^2.1.2",
```

Comment:
Is it possible to separate this PR from the dependency fix so we can discuss them each on their merits?

Reply:
Done. I had to branch off of the other one to get the fix